src/Monolog/Handler/LogglyHandler.php

```diff
@@ -92,11 +92,45 @@ protected function send($data, $endpoint)
         curl_setopt($ch, CURLOPT_HTTPHEADER, $headers);
         curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
 
-        if (curl_exec($ch) === false) {
-            throw new \RuntimeException(sprintf('Curl error (code %s): %s', curl_errno($ch), curl_error($ch)));
-        }
-
-        curl_close($ch);
+        $RETRY_COUNT = 5;  
+        $TOTAL_RETRIES = 0;
+
+        /*
+        * Curl error code 6  : CURLE_COULDNT_RESOLVE_HOST  
+        * Curl error code 7  : CURLE_COULDNT_CONNECT
+        * Curl error code 9  : CURLE_REMOTE_ACCESS_DENIED 
+        * Curl error code 22 : CURLE_HTTP_RETURNED_ERROR
+        * Curl error code 25 : CURLE_UPLOAD_FAILED
+        * Curl error code 26 : CURLE_READ_ERROR 
+        * Curl error code 28 : CURLE_OPERATION_TIMEDOUT
+        * Curl error code 34 : CURLE_HTTP_POST_ERROR 
+        * Curl error code 35 : CURLE_SSL_CONNECT_ERROR 
+        *
+        * Curl Error Codes  : http://curl.haxx.se/libcurl/c/libcurl-errors.html
+        */
+        $CurlErrorCodesForRetries = array(6,7,9,22,25,26,28,34,35);
```

Comment:
Only @Seldaek can tell ;-).

Are all those constants defined in the curl version bundled in 5.3.0?